gamepad/microbit/Kconfig

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
Constants defined in this file are then used in the code. I think it would probably work by setting them directly, but I'd rather just keep it the same as the original sample.

Reply:
Ok, I thought that these were part of the SDK, but apparently they're not.